test/run_test.py

```diff
@@ -70,6 +70,8 @@
     'test_overrides',
     'test_jit_fuser_te',
     'test_tensorexpr',
+    'distributed/nn/jit/test_instantiator',
+    'distributed/nn/api/test_remote_module_spawn',
```

Comment:
should this also be added to `SLOW_TESTS`?

Reply:
Good point! Adding.